### PR TITLE
crate stable release 0.50.3

### PR DESCRIPTION
**Pull request by mikethebeer**

### CODE REVIEW

**Comment by yosifkit** [comment]
I'm getting an `invalid ref` for the commit on `0.50.3`: `c466b48cf0d243f40ba29eba84c2e8336ebcdeb0`.

File: library/crate [hunk 1]
```diff
@@ -7,8 +7,8 @@
 
 # see also https://crate.io
 
-latest: git://github.com/crate/docker-crate@3a44baa2e8179cb0fdb7540e271b2d2f6f3fa90b
+latest: git://github.com/crate/docker-crate@c466b48cf0d243f40ba29eba84c2e8336ebcdeb0
 0.49: git://github.com/crate/docker-crate@8762d54054d9ed9d76a3bba0f9e6fa38acd34c2a
 0.49.6: git://github.com/crate/docker-crate@8762d54054d9ed9d76a3bba0f9e6fa38acd34c2a
 0.50: git://github.com/crate/docker-crate@3a44baa2e8179cb0fdb7540e271b2d2f6f3fa90b
-0.50.2: git://github.com/crate/docker-crate@3a44baa2e8179cb0fdb7540e271b2d2f6f3fa90b
+0.50.3: git://github.com/crate/docker-crate@c466b48cf0d243f40ba29eba84c2e8336ebcdeb0
```
Comment (tianon) [inline library/crate]: Looks like this should've been `5c40f0f6f90d1519f1fda2c6ebe215fdd9d84012` (and `latest` and `0.50` should be too)?

**Comment by mikethebeer** [comment]
hi @yosifkit and @tianon,
yes of course. fatal mistake ^^ i've made a fixup. Ty

**Comment by tianon** [comment]
LGTM (https://github.com/crate/docker-crate/compare/3a44baa2e8179cb0fdb7540e271b2d2f6f3fa90b...5c40f0f6f90d1519f1fda2c6ebe215fdd9d84012)

**Comment by yosifkit** [comment]
LGTM, Build test of #955; 804d7911a8b03ac44f45c968a2aae9978d4b8a46 (`crate`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/804d7911a8b03ac44f45c968a2aae9978d4b8a46/library/crate"
$ bashbrew build "$url"
Processing crate:latest ...
Processing crate:0.49 ...
Processing crate:0.49.6 ...
Processing crate:0.50 ...
Processing crate:0.50.3 ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing crate:latest
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing crate:0.49
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
```

**Pull request merged**